src/scales/scale.time.js

```diff
@@ -176,6 +176,19 @@ module.exports = function(Chart) {
 			Chart.Scale.prototype.initialize.call(this);
 		},
 
+        /**
+         * Allow x-axis to be accessed via x or t attribute
+         */
+		getRightValue: function(rawValue) {
+			if (typeof(rawValue) === 'object') {
+				if ((rawValue instanceof Date) || rawValue.isValid) {
```

Comment:
`helpers.isObject` doesn't handle Date as an object. I'm still thinking that my previous comment is valid: only handle defined `object.x` and `object.y ` else return `rawValue`. If `rawValue` is a date, then it works as before, no need for a special case. Also, it should not call `me.getRightValue` if `rawValue` doesn't define `x` (or `y`) properties.